Software-Engineering/Levels-Maturity-And-Performance-Reviews.md

```diff
@@ -28,19 +30,20 @@ Use these tests to check if someone is ready for a promotion to Maturing at the
 2. _Do you and the person have a shared understanding of the gap to get to Performing at that next level, and are you both up to the challenge to put that plan into action_?
 3. _Have you lined up the opportunities that person will need to demonstrate they can get to Performing at that next level?_
 
-There is no hard and fast rule for a maximum duration that someone can be in Maturing. The important thing is that the manager is confident about the person's growth trajectory toward Performing. If this confidence is lost, the manager and person should discuss moving back to Exceeding in the previous level.
+Even though it's typical for progress to be faster in lower levels and slower in higher levels, **there is no hard and fast rule for a maximum duration that someone can be in Maturing**. The important thing is that the manager is confident about the person's **growth trajectory toward Performing**. If this confidence is lost, the manager and person should discuss moving back to Exceeding in the previous level.
```

Comment:
> Even though it's typical for progress to be faster in lower levels and slower in higher levels

I might have missed some context here... In my experience, I haven't seen a very significant difference across levels and I'm not sure which FAQ this helps to clear up.

I think the most helpful message we can hope to emphasize is that **it's not about tenure** at all, ever. It's about performance with respect to expectations.

Reply:
I'll remove this. I don't want to hamper the more important progress in this proposal.